src/state_manager/mod.rs

```diff
@@ -1313,6 +1314,36 @@ where
         )
     }
 
+    pub fn verified_client_status(
+        self: &Arc<Self>,
+        addr: &Address,
+        ts: &Arc<Tipset>,
+    ) -> anyhow::Result<Option<DataCap>> {
+        let id = self.lookup_id(addr, ts)?.expect("actor not found");
```

Comment:
Shouldn't we rather propagate the error? We wouldn't like to crash the node in case in case of a massing actor, no?

Reply:
Yeah, you are right. We should, I will just convert the `None` value to an error.